.gitignore

```diff
@@ -19,3 +19,10 @@ bundle.integrity.manifest.json
 # yarn
 yarn-debug.log*
 yarn-error.log*
+.yarn/*
```

Comment:
if we are excluding `.yarn/*` should this file be removed `.yarn/plugins/@yarnpkg/plugin-interactive-tools.cjs`?

Reply:
and this ones `.yarn/plugins/@yarnpkg/plugin-workspace-tools.cjs` `.yarn/releases/yarn-2.4.1.cjs`

Reply:
good questions, we are except - patches,releases,plugins,sdks and versions

Reply:
I don't see them explicitly called out in gitignore, the entire `.yarn/*` folder should be ignored so those files probably have been added my mistake?

Reply:
i might not have been clear earlier we are ignoring `.yarn/*` except the following files because of this https://yarnpkg.com/getting-started/qa#which-files-should-be-gitignored a recommendation from Yarn 2. I'm not sure if that answers the question?


> .yarn/patches contain the patchfiles you've been generating with the yarn patch-commit command. You always want them in your repository, since they are necessary to install your dependencies.
> 
> .yarn/plugins and .yarn/releases contain the Yarn releases used in the current repository (as defined by yarn set version). You will want to keep them versioned (this prevents potential issues if, say, two engineers use different Yarn versions with different features).
> 
> .yarn/sdks contains the editor SDKs generated by PnPify. Whether to keep it in your repository or not is up to you; if you don't, you'll need to follow the editor procedure again on new clones. See Editor SDKs for more details.
> 
> .yarn/unplugged should likely always be ignored since they typically hold machine-specific build artifacts. Ignoring it might however prevent Zero-Installs from working (to prevent this, set enableScripts to false).
> 
> .yarn/versions is used by the version plugin to store the package release definitions. You will want to keep it within your repository.

Reply:
i've added this  so that the above files are not ignored and added to the repository excluding other files within the yarn folder
> 
> !.yarn/patches
> !.yarn/releases
> !.yarn/plugins
> !.yarn/sdks
> !.yarn/versions

Reply:
Thanks @nellyk